server/test/article.spec.js

```diff
@@ -174,13 +174,29 @@ describe('Articles routes', () => {
     });
     it('it should get all articles', async () => {
       const res = await chai.request(app)
-        .get('/api/v1/article')
+        .get('/api/v1/article/1')
         .set({ Authorization: `Bearer ${userToken}` });
       const { message } = res.body;
       expect(res).to.have.status(200);
       expect(message).to.be.equal('Articles successfully retrieved');
       expect(res.body.data).to.have.property('articles');
     });
+    it('it should return error if no more articles on page', async () => {
+      const res = await chai.request(app)
+        .get('/api/v1/article/4')
+        .set({ Authorization: `Bearer ${userToken}` });
+      const { status, message } = res.body;
+      expect(message).to.be.equal('There are no articles here');
+      expect(status).to.be.equal(404);
+    });
+    it('it should return error if wrong params is passed', async () => {
+      const res = await chai.request(app)
+        .get('/api/v1/article/g')
+        .set({ Authorization: `Bearer ${userToken}` });
```

Comment:
Kindly look into this too, user do not need to be authenticated to view articles.